src/main/java/org/jenkinsci/plugins/edithelp/EditHelpRootAction.java

```diff
@@ -0,0 +1,155 @@
+package org.jenkinsci.plugins.edithelp;
+
+import java.io.*;
```

Comment:
Please avoid using bulk imports in the production code. It may cause issues in the case of library updates and class conflicts (also invisible issues)